test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Converters/DeprecationToDeprecationLabelStateConverterTests.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using NuGet.Versioning;
+using NuGet.VisualStudio.Internal.Contracts;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test.Converters
+{
+    public class DeprecationToDeprecationLabelStateConverterTests
+    {
+        public static IEnumerable<object[]> GetData()
+        {
+            yield return new object[] { null, PackageItemDeprecationLabelState.Invisible };
+
```

Comment:
Can we remove some of the extra blank lines? Maybe group each declaration with its yield return pair without line breaks between.

Reply:
grouped by yield/value pair.